### PR TITLE
Discovery vs introduction

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Comment by ekr** [comment]
This attempts to enact what we discussed in the room.

**Comment by britram** [comment]
+1, this captures where the room is converging quite well.

File: charter.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The More Instant Messaging Interoperability (MIMI) working group will specify th
...
+Achieving interoperability requires a solution for the introduction problem, i.e., the ability for a user in one application to take an identifier for a target user along with the associated application, be granted permission to initiate communications, and be able to establish communications with the target. The working group will specify a solution to the introduction problem, together with best practice recommendations for functionality, configuration options, and other aspects. The working group may also choose to specify a solution to discovery the set of preferred applications associated with a given identifier. Express and implied user preferences about discoverability and reachability must be respected.
```
Comment (turt2live) [inline charter.md:7]: tiny typo (I think?):
```suggestion
Achieving interoperability requires a solution for the introduction problem, i.e., the ability for a user in one application to take an identifier for a target user along with the associated application, be granted permission to initiate communications, and be able to establish communications with the target. The working group will specify a solution to the introduction problem, together with best practice recommendations for functionality, configuration options, and other aspects. The working group may also choose to specify a solution to discover the set of preferred applications associated with a given identifier. Express and implied user preferences about discoverability and reachability must be respected. 
```

**Review by rohan-wire — APPROVED** [review]

**Comment by jdrosen** [comment]
I'm fine with this. There is a typo on discover vs discovery

**Comment by ekr** [comment]
Not any more :)

**Pull request merged**